src/main/java/com/gtnewhorizons/modularui/common/widget/textfield/BaseTextFieldWidget.java

```diff
@@ -282,7 +287,8 @@ public boolean onKeyPressed(char character, int keyCode) {
             // mark whole text
             handler.markAll();
             return true;
-        } else if (BASE_PATTERN.matcher(String.valueOf(character)).matches()) {
+            // Support CJKV Unified Ideographs
+        } else if (BASE_PATTERN.matcher(String.valueOf(character)).matches() || Character.isIdeographic(character)) {
```

Comment:
I think this check can just be removed, there're lots of non-ascii, non-CJKV characters like Cyrillic

Reply:
Not sure if some special unicode emoji like →❤️✨ can put here